aws/sdk/benchmarks/s3-express/README.md

```diff
@@ -0,0 +1,38 @@
+# S3 Express Benchmark
+
+This directory contains S3 Express One Zone benchmarks that measure end-to-end throughput when using the AWS Rust SDK to put, get, and delete objects to/from S3 Express One Zone buckets. We use [`Criterion`](https://github.com/bheisler/criterion.rs) for benchmarks.
+
+Performance numbers will vary depending on the benchmarking environment, but relative performance should still be accurate (i.e. regular S3 bucket vs. S3 Express bucket or comparing with a previous release of the Rust SDK).
+
+## Benchmark targets
+- `put_get_delete`: `PutObject`, `GetObject`, and `DeleteObject` using sequential invocations (20 by default) of operations across different buckets, switching buckets on every request and using both 64KB and 1MB objects.
```

Comment:
**correctness**: I don't think we want to benchmark all three together in terms of throughput. We should separate it and probably just benchmark `put` vs `get`. 

This is important in particular because S3 express is [advertised](https://aws.amazon.com/s3/storage-classes/express-one-zone/) as

> purpose-built to deliver consistent single-digit millisecond data access 

If we combine all three operations into one in the benchmark it is harder to dive into whether put vs get is having issues on a regression.

Reply:
This combined operations, again, come from the internal benchmark doc. Let me add separate benchmarks, one for `put` and one for `get`.

Reply:
This combined operations, again, come from the internal benchmark doc. Let me add separate benchmarks, one for put and one for get.

Reply:
Added separate benchmarks in 2da2a76. I left the remaining ones; while your assertion about benchmarking combined operations is absolutely correct, I want to leave these as a proof for the internal benchmark doc that we've implemented them.